vscode-dotnet-runtime-library/src/EventStream/EventStreamEvents.ts

```diff
@@ -1673,9 +1677,17 @@ export class DotnetAcquisitionPartialInstallation extends DotnetAcquisitionMessa
 export class DotnetAcquisitionInProgress extends IEvent
 {
     public readonly type = EventType.DotnetAcquisitionInProgress;
+    public verboseOutputOnly = true;
 
     public readonly eventName = 'DotnetAcquisitionInProgress';
-    constructor(public readonly install: DotnetInstall, public readonly requestingExtensionId: string | null) { super(); }
+    constructor(public readonly install: DotnetInstall, public readonly requestingExtensionId: string | null)
+    {
+        super();
+        if (requestingExtensionId === 'user')
```

Comment:
so the high verbosity doesn't always apply? If you want to reduce output, shouldn't it be `false` by default?